### PR TITLE
RAC-6553 Port add & delete volume redfish api

### PR DESCRIPTION
**Pull request by yaolingling**
**Background**
To port add & delete volume redfish api, the parameters of the two graphs need to be added. Three parameters are  ipAddress, username and password, which is used for obm setting. Details in https://rackhd.atlassian.net/browse/RAC-6553.

**Reviewers**
@nortonluo @lanchongyizu @AlaricChan

### CODE REVIEW

File: lib/jobs/dell-wsman-RAID.js [hunk 1]
```diff
@@ -56,10 +56,22 @@ function DellWsmanRAIDFactory(
 
     DellWsmanRAIDJob.prototype._handleSyncRequest = function() {
         var self = this;
-        return self.checkOBM('SCP RAID operation')
-        .then(function(obm){
+        if(self.options.ipAddress === '' || self.options.username === '' ||
+            self.options.password === ''){
```
Comment (pengz1) [inline lib/jobs/dell-wsman-RAID.js]: Your schema doesn't restrict type of those ipAddress/username/password to be string, you should consider cases that those parameters to be null and undefined as well. 
Expression like (Job.Dell.Wsman.RAID || self.options.username ||  self.options.password) should be better for you.

Comment (yaolingling) [reply lib/jobs/dell-wsman-RAID.js]: The type of ipAddress/username/password is string, which defined in schema.

Comment (pengz1) [reply lib/jobs/dell-wsman-RAID.js]: If you are mentioning dell-wsman-RAID.json, I didn't see that.

Comment (pengz1) [reply lib/jobs/dell-wsman-RAID.js]: OK, I see you update schema in this PR as well.

**Pull request merged**